### PR TITLE
Add an image detail display for image tags

### PR DESCRIPTION
**Pull request by brianhelba**

### CODE REVIEW

**Comment by brianhelba** [comment]
@msmolens Can you take a look at this, and see if you can get something decent? This version isn't layout properly. I was trying out `.label.label-default`, but I'm open to other ways of styling.

**Comment by brianhelba** [comment]
@msmolens There's no urgency, the important feature was the faceted search for tags.

**Comment by brianhelba** [comment]
@msmolens Is this ready?

**Comment by msmolens** [comment]
@brianhelba Yes. We could update the first commit to remove "WIP" before removing "WIP" from the PR.

File: web_external/ImagesGallery/Detail/imageDetailsPage.jade [hunk 1]
```diff
@@ -86,9 +86,10 @@
...
         td.isic-image-details-table-section-header(colspan=2) Tags
       tr
-        td
+        td(colspan=2)
           each value in image.get('notes').tags
-            span.label.label-default= value
-        td
+            //- Labels separated by a space
+            span.label.label-default.isic-image-details-tag= value
+            | &#x20;
```
Comment (brianhelba) [inline web_external/ImagesGallery/Detail/imageDetailsPage.jade]: Why not use `&nbsp;`? It's a little more idiomatic for HTML than reaching into Unicode.

Comment (brianhelba) [inline web_external/ImagesGallery/Detail/imageDetailsPage.jade]: This obviously isn't a big deal, but IMO, the `|` often makes rendering harder to control. For mixed variable and literal content, I'd just interpolate the variables into a single line, as:
```
span... #{value} &#x20;
```

**Comment by brianhelba** [comment]
@msmolens See my comments. Also, feel free to rebase and squash your commit(s) on top of my WIP one, since mine provides nothing interesting.

File: web_external/ImagesGallery/Detail/imageDetailsPage.jade (hunk 1 above)
Comment (msmolens) [inline web_external/ImagesGallery/Detail/imageDetailsPage.jade]: - Normally, yes, I'd first reach for `&nbsp;`. But here the intent of the space is to actually break the line, in addition to providing a visual buffer between the labels.
- The space is outside of the span, so it shouldn't be on the same line as the value.

I'll rebase/squash now.

**Comment by brianhelba** [comment]
Looks fine, thanks!

**Pull request merged**